plugins/chat/assets/javascripts/discourse/components/chat-channel-card.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="chat-channel-card__header">
       <LinkTo
         @route="chat.channel"
-        @models={{array @channel.id (slugify-channel @channel)}}
+        @models={{array @channel.slugifiedTitle @channel.id}}
```

Comment:
We could maybe have a `routeModels` methods on `channel` which would return `[slug, id]`? what do you think?